.profile

```diff
@@ -32,11 +38,10 @@ function __gpg_gitconfig () {
     git config --global tag.gpgsign "true" ;
 }
 
-# shellcheck disable=SC2125
 function __gpg_vscode () {
     local VSCODE SETTINGS_JSON
     # ensure a .vscode folder exists
-    VSCODE="${THEIA_WORKSPACE_ROOT:-/workspace/*}/.vscode"
+    VSCODE="/workspace/.vscode-remote"
```

Comment:
Change target `settings.json` file to the root `/workspace/.vscode-remote` folder. Experimental. Ideally this will work better than modifying the active workspace's files (current solution).